### DISCUSSION TITLE
refresh rust-analyzer

### DISCUSSION THREAD

**Discussion by BKSalman** [helix-editor/helix#4386] [Q&A] [answered]
Hello, sometimes I add new dependency crates to the project while having helix open, and I want to refresh rust-analyzer while I'm inside helix, currently I close it and reopen it everytime.

is there a way to refresh it with a command or something?

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/pull/3435

**Reply by BKSalman** [reply]
I see it was merged, but has it been released?

**Reply by the-mikedavis** [reply]
Not yet, it was merged after the 22.08.1 release

**Reply by BKSalman** [reply]
I see, thank you for explaining, and thanks for the hard work!!

**Reply by BKSalman** [reply]
> #3435

thanks for mentioning the PR

**Comment by the-mikedavis** [comment, marked as answer]
With the changes in https://github.com/helix-editor/helix/pull/2653, changes to Cargo.toml would be sent to rust-analyzer and it would be able to download and analyze new dependencies
